views/Login/index.tsx

```diff
@@ -1,51 +1,49 @@
-import { useLayoutEffect } from 'react'
+import { useLayoutEffect as useEffect } from 'react'
 import { NextPage } from 'next'
-import { useRouter } from 'next/router'
 import { useLazyQuery } from '@apollo/client'
-import { toast } from 'react-toastify'
 import { GET_USER_SESSION } from '@/services/GraphQL/queries/users'
 import { LoginInput } from '@/services/GraphQL/types/users'
-import UseError from '@/hooks/useError'
+import useLogger from '@/hooks/useLogger'
+import useMagicLink from '@/hooks/useMagicLink'
 import LoginLayout from './Layout'
 
 const Login: NextPage = () => {
-  const router = useRouter()
   const [checkUserSession, { loading, data }] = useLazyQuery(GET_USER_SESSION)
-  const [createError] = UseError()
+  const { log, warn, error } = useLogger()
+  const magicLink = useMagicLink()
 
-  useLayoutEffect(() => {
-    if (data?.users?.length === 1) {
-      window.sessionStorage.setItem('userId', data.users[0].id)
-      window.sessionStorage.setItem('user', JSON.stringify(data.users[0]))
-      router.push('/eventos/')
-    }
+  useEffect(() => {
+    if (!data) return
     if (data?.users?.length === 0) {
-      toast('Usuario o contraseña incorrectos', { type: 'error' })
+      warn(
+        'Login:useEffect',
+        'No se encontró ningún usuario en base de datos, Usuario o contraseña incorrectos...',
+        'INPUT',
+      )
+    } else {
+      if (data?.users?.length > 1) {
+        warn(
+          'Login:useEffect',
+          'Se encontraron varios usuarios en base de datos, usando el primer email. Informe al servicio técnico del incidente...',
+          'INPUT',
+        )
+      }
+      log('Login:useEffect', 'Usuario encontrado en base de datos, procediendo a verificar sesión...', 'SUCCESS')
+      if (data.users[0]?.is_active && data.users[0]?.member_information?.email) {
+        magicLink(data.users[0])
+      } else {
+        warn('Login:useEffect', 'Usuario no activo, contactar con el administrador...', 'AUTHORIZATION')
+      }
```

Comment:
I think you've missed request is_active from the database 

![image](https://user-images.githubusercontent.com/66505715/172728173-deb107a8-cbc2-4db1-93c0-dbb7c856ba4d.png)

Reply:
I erased the condition but left the else in this condition as a safety net if in the future the DB changes.

Reply:
I think is better to manage the active logic here